spec/file_instance_repo_spec.rb

```diff
@@ -33,6 +40,14 @@ module Fauxpaas
     end
 
     describe "#save_releases" do
+      it "find legacy releases" do
+        instance = static_repo.find("test-legacypersistence")
+        tmp_repo.save_releases(instance)
```

Comment:
Are these making sure it re-stores arrays as separate elements? I'm not having an easy time parsing this, what it's doing, or why it's here. Everything else looks like a pretty straightforward degeneration from arrays to single items, but this part confuses me.

Reply:
Ok I re-read your comment, and it looks like this is under "Adds legacy support for existing instances." I still don't get what it's doing, but I'm happy to assume it's my unfamiliarity with the code base rather than anything particularly confusing about the code.

Which is to say, I understand the titles of these specs, but I don't understand their content.